pipeline/tasks/task-scripts/prepare-site.sh

```diff
@@ -52,6 +52,15 @@ if [[ "${SP_PROTECTED}" != "false" ]] ; then
   echo "  SP_ENTITY_ID: '${spProtocol}://${SP_DOMAIN_NAME}/saml/metadata'" >> ${appFile}
   echo "  SP_PROTECTED: ${SP_PROTECTED}" >> ${appFile}
   echo "  VALID_TOKEN_AGE: ${VALID_TOKEN_AGE}" >> ${appFile}
+  echo "  GATSBY_FEATURE_UPLOADCARE_IS_ENABLED: ${GATSBY_FEATURE_UPLOADCARE_IS_ENABLED}" >> ${appFile}
+  echo "  GATSBY_UPLOADCARE_PUBLIC_KEY: ${GATSBY_UPLOADCARE_PUBLIC_KEY}" >> ${appFile}
+  echo "  ASSET_BUCKET_REGION: ${ASSET_BUCKET_REGION}" >> ${appFile}
+  set +x
+  echo "  UPLOADCARE_SECRET_KEY: ${UPLOADCARE_SECRET_KEY}" >> ${appFile}
+  echo "  UPLOADCARE_STORAGE_ID: ${UPLOADCARE_STORAGE_ID}" >> ${appFile}
+  echo "  UPLOAD_SIGNATURE_EXPIRY_SECONDS: ${UPLOAD_SIGNATURE_EXPIRY_SECONDS}" >> ${appFile}
```

Comment:
This env var has a different name to all the others?

Reply:
changed